src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -149,6 +149,345 @@ Once the server is up, ProActive Web Portals are available at `\http://localhost
 
 Your ProActive Scheduler is ready to execute <<_glossary_job,*Jobs*>>!
 
+
+==== How to install ProActive using Docker Containers
+ProActive could be easily deployed and started within a fully containerized environment. For this purpose, we provide a ready-to-use Docker Compose configuration that enables you running ProActive Workflows and Scheduling as a containerized service.
+It starts the following services:
+
+*  Proactive server container
+*  Proactive node container
+*  Proactive database container (Postgres or Mysql). The database container is not started when using our embedded HSQL database.
+
+The Docker Compose configuration pulls the Proactive server, node, and database images from the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^].
```

Comment:
"," after "node" and before "and" is not required

Reply:
It is https://www.grammarly.com/blog/comma-before-and/